home.css

```diff
@@ -166,6 +166,132 @@ main button {
   width: 20%;
 }
 
+button{
+  border:none;
+  outline: none;
+  cursor: pointer;
+}
+section{
+  max-width: 1000px;
+  margin: auto;
+  text-align: center;
+  padding: 0 1rem;
+}
```

Comment:
Add class or id instead of element

Reply:
I have successfully done the changes, plz do review now!